spring-data-vmware-gemfire/src/main/java/org/springframework/data/gemfire/client/GemfireDataSourcePostProcessor.java

```diff
@@ -159,7 +159,8 @@ Iterable<String> regionNames(ClientCache clientCache) {
 
 			try {
 
-				Object results = execute(clientCache, new GetRegionsFunction());
+				// adding false as an argument due to behavior change in GemFire 10.1
+				Object results = execute(clientCache, new GetRegionsFunction(), false);
```

Comment:
I get that we needed to do this, but why did we not hit this problem before moving to test containers?